src/settings/StaffSlips/components/ViewSections/StaffSlipTemplateContentSection/StaffSlipTemplateContentSection.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+} from 'react-intl';
+import HtmlToReact, { Parser } from 'html-to-react';
+
+import {
+  Button,
+  Col,
+  Row,
+} from '@folio/stripes/components';
+import { PreviewModal, tokensReducer } from '@folio/stripes-template-editor';
+import css from '../../../StaffSlipDetail.css';
+import getTokens from '../../../tokens';
+
+const StaffSlipTemplateContentSection = ({ intl, staffSlip }) => {
+  const {
+    formatMessage,
+    locale,
+  } = intl;
+  const tokens = getTokens(locale);
+  const { name, template } = staffSlip;
+
+  const processNodeDefinitions = new HtmlToReact.ProcessNodeDefinitions(React);
+  const rules = [
+    {
+      shouldProcessNode: () => true,
+      processNode: processNodeDefinitions.processDefaultNode,
+    },
+  ];
+
+  const parser = new Parser();
+  const parsedEmailTemplate = parser.parseWithInstructions(template, () => true, rules);
+
+  const [openPreview, setOpenPreview] = useState(false);
+
+  const togglePreviewDialog = () => {
+    setOpenPreview(!openPreview);
+  };
```

Comment:
Could you remove spaces between constants declaration?

Reply:
Hi @artem-blazhko,
Having empty lines enhances the readability.
It would be good to enforce such things through eslint rules.

Reply:
Hello @Terala-Priyanka,
Agree with you, it would be good to have eslint rule. But I see an inconsistent approach. In some places we have empty lines, in other places we do not have (see screenshot below). 
Let's use one approach. I would suggest removing empty lines, we try to use this approach through `ui-circulation` app.
![image](https://user-images.githubusercontent.com/42437614/220309634-1807d48a-ee40-4eb8-917e-e70975eca8d8.png)